rails_heroicon.gemspec

```diff
@@ -18,6 +18,8 @@ Gem::Specification.new do |spec|
   spec.files         = Dir["lib/**/*", "compressed/**/*", "CHANGELOG.md", "LICENSE.txt", "README.md"]
   spec.require_paths = ["lib"]
 
+  spec.add_runtime_dependency "rails", ["~> 5.0"]
```

Comment:
Can you remove the square brackets because I believe it isn't necessary. Thanks :+1:

Reply:
Totally missed that, will do right away. Thanks!

Reply:
I kinda messed up and deleted the forked repo. I created another PR that fixes the change request, is it possible to merge the PRs together?

Reply:
Yeah sure, I'll look into it.

Reply:
I'll close this and merge the other PR. Thanks for your contribution.

Reply:
Thank you also!